lib/fun_with_flags/config.ex

```diff
@@ -32,6 +32,10 @@ defmodule FunWithFlags.Config do
     end
   end
 
+  def table_name do
+    Application.get_env(:fun_with_flags, :table_name, "fun_with_flags_toggles")
```

Comment:
I would put `"fun_with_flags_toggles"` in a module attribute:

```elixir
@default_ecto_table_name "fun_with_flags_toggles"
```

Reply:
Also, the concept of `table_name` should really be namespaced under the `:persistence` group, and clearly reference ecto. So this could be:

```elixir
@default_persistence_config [
  adapter: FunWithFlags.Store.Persistent.Redis,
  repo: FunWithFlags.NullEctoRepo,
  ecto_table_name: @default_ecto_table_name
]

# ...
def ecto_table_name do
  Keyword.get(persistence_config(), :ecto_table_name)
end
```

And then be configured with:

```elixir
config :fun_with_flags, :persistence,
  adapter: FunWithFlags.Store.Persistent.Ecto,
  repo: FunWithFlags.Dev.EctoRepo,
  ecto_table_name: "my_table"
```

Reply:
I know that `repo` is not namespaced as `ecto_repo`, but:

1. Changing that now would be a big breaking change.
2. In a Phoenix and/or Ecto app, "repo" is almost universally understood as an Ecto repo. On the other hand "table" could be an ETS table, or some people might think it affects Redis somehow. I'd rather avoid ambiguities.